src/libstd/keyword_docs.rs

```diff
@@ -1578,12 +1578,108 @@ mod type_keyword {}
 
 #[doc(keyword = "unsafe")]
 //
-/// Code or interfaces whose [memory safety] cannot be verified by the type system.
+/// Code or interfaces whose [memory safety] cannot be verified by the type
+/// system.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// The `unsafe` keyword has two uses: to declare the existence of contracts the
+/// compiler can't check, and to declare that a programmer has checked that
+/// these contracts have been upheld.
+///
+/// Unsafe Rust add some capabilities to Safe Rust:
+///
+/// - Dereference [raw pointers]
+/// - Call `unsafe` functions (including foreign functions, compiler intrinsics,
+/// and the raw allocator)
+/// - Implement `unsafe` [`trait`]s
+/// - Mutate [`static`]s (including [`extern`]al ones)
+/// - Access fields of [`union`]s
```

Comment:
This is from https://doc.rust-lang.org/nomicon/what-unsafe-does.html, right? Would it be a good idea to link there? Or have that page link here?

Reply:
The link is available at the bottom as `[Rustnomicon]`. I can add it here, though I'm not sure on how ?

Reply:
Hmm, maybe make the overview a link?

```suggestion
/// Unsafe Rust [adds some capabilities][Rustnomicon] to Safe Rust:
///
/// - Dereference [raw pointers]
/// - Call `unsafe` functions (including foreign functions, compiler intrinsics,
/// and the raw allocator)
/// - Implement `unsafe` [`trait`]s
/// - Mutate [`static`]s (including [`extern`]al ones)
/// - Access fields of [`union`]s
```